fred/cogs/crashes.py

```diff
@@ -300,16 +300,34 @@ async def process_file(self, file, extension) -> list[dict]():
                     return []
 
     async def mass_regex(self, text: str) -> AsyncIterator[dict]:
-        for crash in config.Crashes.fetch_all():
-            if match := await regex_with_timeout(crash["crash"], text, flags=re.IGNORECASE):
+
+        async def silly_async_gen():
+            nonlocal text
+            all_crashes = config.Crashes.fetch_all()
+            for crash_ in all_crashes:
+                yield crash_, regex_with_timeout(crash_["crash"], text, flags=regex.IGNORECASE)
+
+        async def patience(coro):
```

Comment:
If we keep those (see below) please rename to something more communicative lol

Reply:
lol yeah, it was 3am and I forgot to change those